runtime/src/lib.rs

```diff
@@ -219,31 +223,34 @@ parameter_types! {
 	pub const Version: RuntimeVersion = VERSION;
 	pub const BlockHashCount: BlockNumber = 2400;
 	/// We allow for 2 seconds of compute with a 6 second average block time.
-	pub BlockWeights: frame_system::limits::BlockWeights = frame_system::limits::BlockWeights
-		::with_sensible_defaults(WEIGHT_PER_SECOND.saturating_mul(2), NORMAL_DISPATCH_RATIO);
+	pub BlockWeights: frame_system::limits::BlockWeights =
+		frame_system::limits::BlockWeights::with_sensible_defaults(
+			(2u64 * WEIGHT_PER_SECOND).set_proof_size(u64::MAX),
+			NORMAL_DISPATCH_RATIO,
+		);
```

Comment:
How did you find our what you need to do to solve this? I am now having the same issue on the parachain, but my weight declaration is the same as statemines

Reply:
Maybe helps:

https://substrate.stackexchange.com/questions/5557/construct-runtime-integrity-test-failing

https://github.com/paritytech/substrate/pull/12277

https://github.com/substrate-developer-hub/substrate-node-template/commit/378a3e5ae41df1af0ef1a66b2076342bce678688#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ec